src/js/controller/controller.js

```diff
@@ -263,7 +264,7 @@ define([
 
                 _checkAutoStart();
 
-                _model.set('preloaded', false);
+                _preloaded = false;
```

Comment:
`_preloaded` should be set to `false` inside `stop`. Since it's set to false on `setup` it does not need to be set to `false` again here.